quality/errorprone/android.gradle

```diff
@@ -1,10 +1,16 @@
 apply plugin: 'net.ltgt.errorprone'
 
 dependencies {
-  annotationProcessor "com.uber.nullaway:nullaway:0.4.6"
-  testAnnotationProcessor "com.uber.nullaway:nullaway:0.4.6"
+  if(JavaVersion.current().getMajorVersion().toInteger() < 9) {
```

Comment:
`if (!JavaVersion.current().java9Compatible)`

Reply:
there's 0 javadoc on these `javaXCompatible` methods
https://docs.gradle.org/current/javadoc/org/gradle/api/JavaVersion.html#isJava8Compatible-- (except for java 11)
but looking at their code it looks like you are right
https://github.com/gradle/gradle/blob/master/subprojects/base-services/src/main/java/org/gradle/api/JavaVersion.java#L164-L195